.github/workflows/whitesource.yml

```
@@ -0,0 +1,33 @@
+name: Whitesource Unified Agent Scan - GO
+
+on:
+  pull_request:
+    branches: [ master* ]
```

```suggestion
    branches:
    - master
```

Unless I'm missing something, you don't need the `*`